docker/Makefile

```diff
@@ -26,7 +26,7 @@ localnet:
 
 
 calibration-up: calibration-down
-	LOTUS_IMAGE_TAG=ntwk-calibration-8.8.0 \
+	LOTUS_IMAGE_TAG=ntwk-calibration-8.13.1-f094273f \
```

Comment:
There's not tag for a hot-fix that solves some problems in big file retrievals.
I kind of made a custom version to provide best possible experience in the network.